std/format.d

```diff
@@ -1120,8 +1121,8 @@ struct FormatSpec(Char)
             {
                 if (trailing.ptr[0] == ' ')
                 {
-                    while (!r.empty && std.ascii.isWhite(r.front)) r.popFront();
-                    //r = std.algorithm.find!(not!(std.ascii.isWhite))(r);
```

Comment:
> That's easily fixed, just change isWhite to being selectively imported from std.utf.

I don't know enough about what this code is doing to whether it should be importing `std.uni.isWhite` or `std.ascii.isWhite`. My concern is that with those imports, it's easy to mix them up when refactoring if they're not explicit, and often tests won't catch the difference, because it tends to be subtle. So, it definitely works to not use their FQNs, but it seems error-prone from a maintenance and clarity standpoint (especially if the import isn't right above where it's used).